core/manifest.py

```diff
@@ -0,0 +1,21 @@
+# Autonomy Controls
+ENABLE_AUTONOMY_ID     = 11100
+DISABLE_AUTONOMY_ID    = 11101
+ADD_WAYPOINT_ID        = 11102
+CLEAR_WAYPOINTS_ID     = 11103
+WAYPOINT_REACHED_ID    = 11104
+
+# Drive Board Ids
+DRIVE_BOARD_IP = "134"
+DRIVE_DATA_ID = 1000
+
+# Lighting Board Ids
+LIGHTING_BOARD_IP = "142"
+NOTIFY_ID = 7000
+
+# Nav Board Ids
+NAV_IP_ADDRESS = "136"
```

Comment:
inconsistent naming convention
```suggestion
NAV_BOARD_IP = "136"
```

Reply:
I need to do another manifest pass through for the simulator so I will save this for then.